crates/rooch/src/commands/move_cli/commands/new.rs

```diff
@@ -22,7 +22,7 @@ pub struct New {
     pub new: new::New,
 
     /// Path of the new package to create.
-    #[clap(long = "path", short = 'p', global = true, parse(from_os_str))]
+    #[clap(long = "mpath", short = 'm', global = true, parse(from_os_str))]
```

Comment:
This is just a temporary fix because there were two conflicting `--path` reported before. We should find the real reason for the conflict.

Reply:
Okay, I have raised an issue: https://github.com/rooch-network/rooch/issues/173